src/Admin/Pool.php

```diff
@@ -13,8 +13,8 @@
 
 namespace Sonata\AdminBundle\Admin;
 
+use Psr\Container\ContainerInterface;
```

Comment:
No composer change?

Reply:
👍 I thought we had it because it was already used in https://github.com/sonata-project/SonataAdminBundle/blob/515ba338fc5e853878502925b815ff86d7fdd6df/src/Controller/CRUDController.php#L17.